makefile

```diff
@@ -11,7 +11,7 @@ ifeq ($(PLATFORM),Darwin)
 endif
 ifeq ($(PLATFORM),Linux)
 	THIRD_PARTY_DIR := third_party_libs/linux
-	SYSTEM_LIBS := -lc -lpthread -lrt -lboost_system -lboost_thread-mt
+	SYSTEM_LIBS := -lc -lpthread -lrt -lboost_system -lboost_thread
```

Comment:
Isn't boost_thread equal to boost_thread-mt?  I'm not objecting to this change, I just want to know.